src/pages/groupComparison/GroupComparisonTabs.ts

```diff
@@ -0,0 +1,9 @@
+export enum GroupComparisonTab {
```

Comment:
I could use some help here. I had to pull these enums into their own tabs in order to get the imports to work correctly in `routes.jsx`. When I just imported them in their respective files, the app compiled, but failed at runtime with the following message
```
utilities.js:38 Uncaught TypeError: Cannot read property 'appendChild' of null
    at Object../node_modules/units-css/lib/utilities.js.utilities.getCreatedElementDimensions (utilities.js:38)
    at Object../node_modules/units-css/lib/utilities.js.utilities.getCreatedElementWidth (utilities.js:51)
    at utilities.js:114
    at Object../node_modules/units-css/lib/utilities.js (utilities.js:117)
    at __webpack_require__ (bootstrap:766)
    at fn (bootstrap:129)
    at Object../node_modules/units-css/lib/conversions/length.js (length.js:6)
    at __webpack_require__ (bootstrap:766)
    at fn (bootstrap:129)
    at Object../node_modules/units-css/lib/conversions/index.js (index.js:8)
```